### PR TITLE
Corpora/csip/metadata/amdSec/csip32

### PR DESCRIPTION
**Pull request by andersbonielsen**
Ready for review. @carlwilson or @ltaimre can look at the structMap part and @PhillipTommerholt for the amdSec part.

### CODE REVIEW

File: corpora/csip/metadata/amdsec/CSIP32/test-case.xml [hunk 1]
```diff
@@ -1,23 +1,88 @@
...
+      <!-- Text description of the validation rule -->
+      <description>There should be an XML element named digiprovMD in the mets.xml</description>
+      <!-- Details of the validation error generated by the message, can be one of: [ ERROR | WARNING | INFO ]-->
+      <error level="WARNING">
+        <!-- Expected error message when the rule is violated. -->
+        <message>The element mets/amdSec/digiprovMD does not exist in METS.xml. It SHOULD be there.</message>
+      </error>
+      <corpusPackages>
+        <!-- Details of corpus packages designed to test this validation rule. -->
+        <package name="IP_18000_CSIP32_1" isValid="FALSE">
```
Comment (PhillipTommerholt) [inline corpora/csip/metadata/amdsec/CSIP32/test-case.xml]: isValid should be TRUE according to the new definition in the schema

File: corpora/csip/metadata/amdsec/CSIP32/test-case.xml [hunk 2]
```diff
@@ -1,23 +1,88 @@
...
+      <description>When one or more mets/amdSec/digiprovMD element(s) exist(s) in mets.xml there SHOULD be files in the folder metadata/preservation.
+        The mets schema allows no files to be present since it allows no href to be present in digiprovMD</description>
+      <!-- Details of the validation error generated by the message, can be one of: [ ERROR | WARNING | INFO ]-->
+      <error level="WARNING">
+        <!-- Expected error message when the rule is violated. -->
+        <message>One or more mets/amdSec/digiprovMD elements exist(s). No file(s) found in metadata/preservation. File(s) SHOULD be there.</message>
+      </error>
+      <corpusPackages>
+        <!-- Details of corpus packages designed to test this validation rule. -->
+        <package name="IP_18000_CSIP32_2" isValid="FALSE">
```
Comment (PhillipTommerholt) [inline corpora/csip/metadata/amdsec/CSIP32/test-case.xml]: isValid should be TRUE according to the new definition in the schema for warnings

File: corpora/csip/metadata/amdsec/CSIP32/IP_18000_CSIP32_3/METS.xml [hunk 3]
```diff
@@ -0,0 +1,102 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<!-- Minimal IP with schemas and no dmdSec but a file in the folder metadata/descriptive-->
+<!-- In this example IDs are carring information to ease understanding - they look like paths to ease understanding, but are just IDs, similar to the naming of namespaces - these IDs can be replaced with information less UUID -->
+<!-- CSIPSTR15 goes like this: We recommend including all schema documents for any structured metadata within package. These schema documents SHOULD be placed in a sub-folder called schemas within the Information Package root folder and/or the representation folder. -->
+<mets xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://www.loc.gov/METS/"
+  xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:csip="https://DILCIS.eu/XML/METS/CSIPExtensionMETS"
+  xsi:schemaLocation="http://www.w3.org/2001/XMLSchema-instance schemas/XMLSchema.xsd http://www.loc.gov/METS/ schemas/mets.xsd 
+    http://www.w3.org/1999/xlink schemas/xlink.xsd 
+    https://DILCIS.eu/XML/METS/CSIPExtensionMETS schemas/CSIPExtensionMETS.xsd"
+  OBJID="IP_18000_CSIP17_3" TYPE="Database" csip:CONTENTINFORMATIONTYPE="SIARD2"
```
Comment (PhillipTommerholt) [inline corpora/csip/metadata/amdsec/CSIP32/IP_18000_CSIP32_3/METS.xml]: wrong OBJID

**Review by PhillipTommerholt — CHANGES_REQUESTED** [review]
Made some few comments. Also: rename test-case.xml to testCase.xml

**Review by PhillipTommerholt — DISMISSED** [review]

**Review by carlwilson — APPROVED** [review]

**Pull request merged**